spec/views/user/user_spec.rb

```diff
@@ -0,0 +1,25 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+
+
+RSpec.describe UserController, type: :controller do
```

Comment:
This is file is under the `spec/views` folder, and thus it should not test the `UserController`, this context should be moved to `spec/controllers/user_controller_spec.rb`.

Reply:
Moved to controllers folder, I originally had it testing the view but realised I should do it this way, which is why it was in that folder.